_toc.yml

```diff
@@ -1,42 +1,62 @@
 - file: index
 - chapters:
-  - file: community/index
-    sections:
-    - file: community/mission_and_values
-    - file: community/team
-    - file: community/code_of_conduct
-    - file: community/code_of_conduct_reporting
-    - file: community/governance
-    title: Community
+  - file: tutorials/start_index
+    sections: 
+    - file: tutorials/fundamentals/installation
+    - file: tutorials/absolute_beginners
```

Comment:
What content is the absolute beginners page supposed to cover? (Apologies if this was already discussed elsewhere)

Reply:
I was thinking of an absolute beginners page as a placeholder for the "Intro for non-Python users". There's no content for this yet, but I wanted to make sure this was in the TOC for this PR just so the entire context is captured.